endstate_correction/protocol.py

```diff
@@ -190,7 +190,7 @@ def perform_endstate_correction(protocol: Protocol) -> Results:
             # from target to reference potential
             dEs, _, _ = perform_switching(
                 sim,
-                lambdas=list_of_lambda_values.flip(), # NOTE: we reverse the list of provided lamba values to indicate switching from the reference to the target potential
+                lambdas=np.flip(list_of_lambda_values), # NOTE: we reverse the list of provided lamba values to indicate switching from the reference to the target potential
```

Comment:
I think `np.flip` is working like that..

Reply:
Thanks for that, you are absolutly right!